rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/TestResultFileWriter.xtend

```diff
@@ -75,9 +75,10 @@ class TestResultFileWriter {
 			      failed to execute test, please check your technical test setup 
 			    </error>
 			  </testcase>
-			</testsuite>''', file, StandardCharsets.UTF_8);
+			</testsuite>'''
+			Files.write(content, file, StandardCharsets.UTF_8);
```

Comment:
if you do a static import the call gets even nicer `Files.write(content, file, UTF_8)`